src/components/ConfirmDialogWindow.jsx

```diff
@@ -0,0 +1,72 @@
+import { useA11yDialog } from 'react-a11y-dialog';
+import { createPortal } from 'react-dom';
+import './ConfirmDialogWindow.css';
+
+const ConfirmDialogWindow = ({ text, title, confirmAction }) => {
+	// `instance` is the `a11y-dialog` instance.
+	// `attr` is an object with the following keys:
+	// - `container`: the dialog container
+	// - `overlay`: the dialog overlay (sometimes called backdrop)
+	// - `dialog`: the actual dialog box
+	// - `title`: the dialog mandatory title
+	// - `closeButton`:  the dialog close button
```

Comment:
Great use of comments! Using `//` is great, but just FYI you can use this syntax as well to avoid multiple `//`

```suggestion
        /**
	   `instance` is the `a11y-dialog` instance.
	   `attr` is an object with the following keys:
	   - `container`: the dialog container
	   - `overlay`: the dialog overlay (sometimes called backdrop)
	   - `dialog`: the actual dialog box
	   - `title`: the dialog mandatory title
	   - `closeButton`:  the dialog close button
        */
```

Reply:
I was leftover code from the a11y-dialog documentation, I'd better remove it